gandalf.js

```diff
@@ -1,56 +1,62 @@
 /**
```

Comment:
SCRIPT-This line seems to be longer than our standard line length. Could you split it up for better readability?